Description: Add permission to bluetooth control for user into bluetooth 
Forwarded: not-needed
Origin: vendor
Author: Nobuhiro Iwamatsu <iwamatsu@debian.org>

Index: bluez-5.33/src/bluetooth.conf
===================================================================
--- bluez-5.33.orig/src/bluetooth.conf
+++ bluez-5.33/src/bluetooth.conf
@@ -24,6 +24,11 @@
     <allow send_interface="org.freedesktop.DBus.Properties"/>
   </policy>
 
+  <!-- allow users of bluetooth group to communicate -->
+  <policy group="bluetooth">
+    <allow send_destination="org.bluez"/>
+  </policy>
+
   <policy at_console="true">
     <allow send_destination="org.bluez"/>
   </policy>
